File: bookingservice/src/main/java/com/microservice/bookingservice/model/Slot.java
package com.microservice.bookingservice.model;

import com.fasterxml.jackson.annotation.JsonBackReference;
import javax.persistence.*;

@Entity
public class Slot {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String time;
    private int vacancy;

    @ManyToOne
    @JoinColumn(name = "schedule_date_id")
    @JsonBackReference
    private ScheduleDate scheduleDate;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }

    public int getVacancy() {
        return vacancy;
    }

    public void setVacancy(int vacancy) {
        this.vacancy = vacancy;
    }

    public ScheduleDate getScheduleDate() {
        return scheduleDate;
    }

    public void setScheduleDate(ScheduleDate scheduleDate) {
        this.scheduleDate = scheduleDate;
    }
}

File: Accomodation_service/WebApplication1/Models/Slot.cs
using System;
using System.Collections.Generic;

namespace WebApplication1.Models
{
    public partial class Slot
    {
        public long Id { get; set; }
        public string? Time { get; set; }
        public int Vacancy { get; set; }
        public long? ScheduleDateId { get; set; }

        public virtual ScheduleDate? ScheduleDate { get; set; }
    }
}
